### ISSUE TITLE
Fix remove for tricontourf

### ISSUE THREAD

**Issue by banesullivan** [banesullivan/pyvista-mpl-slicer#1]
https://github.com/banesullivan/pyvista-mpl-slicer/blob/0277de9164eb18dcc07790040bca84939d36a3d9/slicer.py#L376

tricontourf plots do not have a remove method, so right now plots are stacked on top of each other endlessly which is bad

**Comment by prisae** [comment]
Does it matter whether you remove it, or replace it in-place?

**Comment by banesullivan** [comment]
I don’t think it matters, but currently it is stacking them all on top of each other... I think... if you use `tricontour` instead you can see it

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
